tests/conftest.py

```diff
@@ -29,6 +29,12 @@ def rng():
     return np.random.default_rng(seed=42)
 
 
+@pytest.fixture
+def mimic_2():
```

Comment:
Can your test not work with 

```
@pytest.fixture
def mimic_2_10():
    mimic_2_10 = ep.dt.mimic_2()[:10]

    return mimic_2_10
```

?

Reply:
using the smaller set we get the follwoing error:

ConvergenceError: Convergence halted due to matrix inversion problems. Suspicion is high collinearity. Please see the following tips in the lifelines documentation: [https://lifelines.readthe](https://lifelines.readthedocs.io/en/latest/Examples.html#problems-with-convergence-in-the-cox-proportional-hazard-modelMatrix)

so i took the whole mimic_2 set

Reply:
I see, makes sense. I think it's okay if we use the smaller subset and filter the warning here https://github.com/theislab/ehrapy/blob/main/pyproject.toml#L132